### DISCUSSION TITLE
RE: Trial: Using Discussion Forums and Tickets at GitHub replacing SourceForge

### DISCUSSION THREAD

**Discussion by EdoFro** [freeplane/freeplane#256] [General]
Hi all, Hi @dpolivaev,
I'm posting this here because I can't reply in the SourceForge forum to [this topic](https://sourceforge.net/p/freeplane/discussion/758437/thread/20baf2dcb7/)

I'm not been able to:

- create new Topics (That's OK)
- respond to existing topics (that's not so cool)

I'm still allowed to:

- edit my old topics (don't know if this can be helpful)
- mark with an emoticon any post (like in the following image)

![image](https://user-images.githubusercontent.com/35700575/157747957-6be8216d-456f-4415-bfdc-319057dc888f.png)

bye,

edo

**Comment by EdoFro** [comment]
> [Dimitry Polivaev ](https://sourceforge.net/u/dpolivaev/) posted [this in sourceforge](https://sourceforge.net/p/freeplane/discussion/758437/thread/20baf2dcb7/#f26e)- 9 hours ago 
I activated posting to this forum again so that all current discussions can be finished,
but I would like to ask you to start all new topics at the GitHub project discussion page https://github.com/freeplane/freeplane/discussions

Sourceforge forum is not active (or I'm not allowed to participate)

**Reply by dpolivaev** [reply]
Could you please check if it is fixed now?

**Comment by dpolivaev** [comment]
Could you please check if it fixed now?

**Reply by EdoFro** [reply]
Yes it is. Thanks!!
